core/spring-cloud-stream/src/test/java/org/springframework/cloud/stream/function/PollableSourceTests.java

```diff
@@ -42,18 +46,23 @@ void test() {
 				.getCompleteConfiguration(PollableAppSampleConfiguration.class))
 						.web(WebApplicationType.NONE).run(
 								"--spring.jmx.enabled=false",
-								"--spring.cloud.stream.pollable-source=blah")) {
+								"--spring.cloud.stream.pollable-source=blah",
+								"--spring.integration.poller.cron=*/2 * * * * *")) {
```

Comment:
Wdyt about adding tests that cover the other exceptional env properties such as `fixed-rate`, `fixed-delay` and also the case for `max-messages-per-poll`?